src/org/zaproxy/zap/extension/accessControl/resources/Messages.properties

```diff
@@ -28,6 +28,8 @@ accessControl.scanOptions.label.users			= Users to scan as (at least 1):
 accessControl.scanOptions.label.raiseAlerts		= Raise alerts for identified issues:
 accessControl.scanOptions.label.alertsRisk		= Risk level for raised alerts:
 accessControl.scanOptions.button.scan			= Start
+accessControl.scanOptions.error.mode.safe = Access control testing is not allowed in 'Safe' mode.
+accessControl.scanOptions.error.mode.protected = The following context is not allowed in 'Protected' mode:\n{0}
```

Comment:
What would be displayed here if the item simply isn't part of a context? (Default Context?)

Reply:
It must be, the dialogue just allows to select contexts.

Reply:
ok